RX interruption catched
TASK_ALLOCATION_SEND
Got slot: 0 
Task total size (txt+bss): 3124 
Task addr: 2147522956 
Task slot: 0 
Task slot TaskAddr: 2147522956 
RX interruption catched
TASK_ALLOCATION_FINISHED
SendingQueue_front: 0 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um TASK_START!
RX interruption catched
Chegou um TASK_RUN!
Starting Task 3  from app 1 
taskSlot run: 0 
SendingQueue_front: 1 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
Task P3 INIT
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 3 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 4 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 5 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 3 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 2000350 
TotalEnergy: 19882455 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 131098 
TotalEnergy: 9587131 
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 713464 
TotalEnergy: 32852843 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 969095 
TotalEnergy: 42940096 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000001 
TotalEnergy: 44169830 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999823 
TotalEnergy: 44171320 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999972 
TotalEnergy: 44173738 
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send failed - NI_TX occupied!
TX interruption catched
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1052117 
TotalEnergy: 46469509 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 959904 
TotalEnergy: 42594591 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 987679 
TotalEnergy: 43457871 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000140 
TotalEnergy: 43954630 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 44174060 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 44175062 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 44176805 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1040805 
TotalEnergy: 46029062 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 971418 
TotalEnergy: 43048368 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 987679 
TotalEnergy: 43692771 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 1000073 
TotalEnergy: 44178062 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 1 
from app: 1 
SendingQueue_front: 0 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 1 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 2 
from app: 1 
SendingQueue_front: 2 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 3 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 3 
from app: 1 
SendingQueue_front: 4 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999900 
TotalEnergy: 44169196 
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 5 
API_Try2Send success!
TX interruption catched
API_Try2Send failed - empty SendQueue!
taskSlot: 0 
Adding a msg to task 5  in the PIPE slot 0 
from app: 1 
SendingQueue_front: 6 
API_Try2Send success!
Esperando Mensagem!
TX interruption catched
API_Try2Send failed - empty SendQueue!
RX interruption catched
Tem uma mensagem chegando...
RX interruption catched
Mensagem Recebida!
RX interruption catched
RX interruption catched
Chegou um message request! App: 1 Task: 5 
Mensagem encontrada no pipe!
SendingQueue_front: 7 
API_Try2Send success!
dynamicEnergy_Router >> 3953125 
total instructions~~~> 999923 
TotalEnergy: 43714839 
